dialog/information.go

```diff
@@ -7,22 +7,20 @@ import (
 	"fyne.io/fyne/v2/widget"
 )
 
-func createTextDialog(title, message string, icon fyne.Resource, parent fyne.Window) Dialog {
-	d := newDialog(title, message, icon, nil, parent)
-
+func createInformationDialog(title, message string, icon fyne.Resource, parent fyne.Window) Dialog {
+	d := newTextDialog(title, message, theme.ErrorIcon(), parent)
```

Comment:
It seems like this shouldn't ignore `icon` and pass `theme.ErrorIcon()` for everything?

Reply:
Good catch!